lib/src/base-styles/_nx-tile.scss

```diff
@@ -135,11 +135,19 @@ $nx-tile-background-color: $nx-white;
 }
 
 // On pages with tiles with multiple sections these are lighter weight titles
-.nx-tile__section-header {
-  border-top: 1px solid #b7bfdb;
-  margin-top: $nx-spacing-l;
-  margin-bottom: $nx-spacing-md;
-  padding-top: $nx-spacing-l;
+.nx-tile-subsection {
+  .nx-h3--tile-subsection {
```

Comment:
Why the change of this classname?

Reply:
> > Why the change of this classname?
> 
> My thinking was that the changes expressed in the SCSS were modifications to `nx-h3` so the class should reflect that. Though this may be moot with the suggested changes to `<section>`

As I suspected there's no need for the custom classes anymore: https://github.com/sonatype/sonatype-react-shared-components/pull/64/commits/d0be78e77757fc08057408e161bfdde678a3e8d0